packages/api/package.json

```diff
@@ -18,7 +18,7 @@
     "integration": "NODE_ENV=integration node src/server.js",
     "migrate:testing:db": "NODE_ENV=test knex migrate:latest --env=test --knexfile=./.knex/knexfile.js",
     "migrate:dev:db": "knex migrate:latest --knexfile=./.knex/knexfile.js",
-    "migrate:make:dev": "knex migrate:latest --knexfile=./.knex/knexfile.js",
+    "migrate:make:dev": "knex migrate:make --knexfile=./.knex/knexfile.js",
```

Comment:
`migrate:make:dev` was the exact same thing as `migrate:dev:db` and I assumed this was a mistake. Looked up in knex documentation how to create a new migration file